documentation/api/operations-get.md

```diff
@@ -49,7 +49,11 @@ Content-Type: application/json
     "error": null,
     "operationId": "67f07e40-5cca-4709-9062-26302c484f18",
     "createdDateTime": "2021-07-21T06:21:15.315861Z",
-    "status": "Succeeded"  
+    "status": "Succeeded",
+    "processInfo":
```

Comment:
This example and the other one need to be fixed:
- The property is named `process`
- Please format the inner object so that there is only one property per line.
- The inner object is not valid JSON (missing comma between the `"pid":1` and `"uid":"..."`)